examples/hardcoded-passwords.py

```diff
@@ -1,29 +1,58 @@
+# Possible hardcoded password: 'Admin'
+# Severity: Low   Confidence: Medium
 def someFunction(user, password="Admin"):
     print("Hi " + user)
 
 def someFunction2(password):
+    # Possible hardcoded password: 'root'
+    # Severity: Low   Confidence: Medium
```

Comment:
the way comments are done here is inconsistent (indent & before/after)
maybe these should be docstrings?

Reply:
I don't think it matters that much, so long as there's context around what each one is attempting to warn about. With the exception of the first function here, functions without a real body (e.g., using `pass`) have the comment outside the function definition and the rest have the comments before (especially since they're call sites). Personally, given the context of this file, I don't care enough. If Eric _wants_ to change that, it's fine that way too. I don't think this is significant enough to warrant him making a second pass though